_layouts/default.html

```diff
@@ -206,7 +207,7 @@
 
                 carouselContainer.slick({
                   arrows: false,
-                  autoplay: true,
+                  autoplay: false,
```

Comment:
Did you mean to check this change in? Or is it an accidental inclusion from testing?